docs/onboard-public-officers-using-non-se-machines.md

```diff
@@ -1,37 +1,109 @@
+
+  <kbd>![sign-up-submit](assets/images/onboarding/po-non-se/latest-po-sign-up-non-se-gsib-1.png)</kbd>
+
+  4. Click **Submit**. We will send you the TechPass onboarding invitation email.
+
+  > **Additional information**:
+  >- When your request is approved, you'll receive two separate onboarding emails for TechPass and SEED.
+  >- If you have requested for SEED provisioning, you will receive the SEED onboarding email around the next three business days.
+  >- A TechPass account is created for you and will be *pending* to be activated.
+  >- It becomes activated when you [accept the invitation](#step-3-accept-techpass-invitation) and [complete the TechPass onboarding journey](#step-4-onboard-to-techpass).
+  >- The TechPass and SEED onboarding invitation emails are valid for 30 days. Refer to SEED documentation for more information on what to do if your SEED onboarding invitation has expired.
+  >- If you do not onboard to TechPass within 30 days, we will terminate your TechPass account and notify you via email before the termination. You can again sign up via the TechPass portal or request for TechPass and SEED.
+  >- Onboard to TechPass before enrolling your non-GSIB or non-DWP device to SEED.
+  >- If needed, [Set up security verification for WOG account](#step-2-set-up-security-verification-for-the-wog-account).
+
+
+
+  </details>
+
+## Step 2. Set up security verification for the WOG account
 
 <details>
   <summary style="font-size:20px;font-weight:bold">How to set up security verification for WOG account?</summary>
 
   > **Important**:<br>
-  > Public officers need to set up security verification(multi-factor authentication) for their Whole-of-Government(WOG) account if:
-    >- They need to access Singapore Government Technology Stack (SGTS) services and tools from their GMD(internet device onboarded to SEED) device
-    >- Their SG Govt M365 profile is not displayed on their Microsoft Authenticator app.<br><br>
-  > Others, can skip this and proceed to **Step 2. Sign up for TechPass**.
+  > You need to set up security verification (multi-factor authentication) for your Whole-of-Government(WOG) account if:
+    >- You must access Singapore Government Technology Stack (SGTS) services and tools from your GMD device.
+    >- To view your SG Govt M365 profile on the Microsoft Authenticator app.
+  >
+  >  
+  > Others can skip this step and proceed to [**Step 3. Accept TechPass invitation**](#step-3-accept-techpass-invitation).
 
-  _To set up security verification for WOG account_
+  **To set up security verification for WOG account**
 
   1. From your non-SE GSIB device, go to [Azure Active Directory](https://account.activedirectory.windowsazure.com/proofup.aspx).
 
-  ?> If you are prompted to sign in, use your organisation email address and GSIB device password.
+> **Note**: To sign in, use your organisational email address and GSIB device password.
+<!-- is it correct to say GSIB password, or should it be WOG email password-->
```

Comment:
they are the same... i think better to use email password. (in case, some don't have gsib. unlikely but just in case)

Reply:
done, thanks!